main.tex

```diff
@@ -1002,7 +1009,8 @@ \subsubsection*{Provenance of individual \wnt{} simulation models}
 \end{figure}
 
 
-Sometimes, simulation or wet-lab experiments have been conducted, however, the corresponding \SD{} or \WD{} is not shown, but briefly described, and are, thus, without references in the provenance graphs.
+Sometimes, simulation or wet-lab experiments have been conducted, however, the corresponding \SD{} or \WD{} is not shown. Instead, they are briefly described, and thus are without references in the provenance graphs.
```

Comment:
Reducing the sentence complexity. The commas around "thus" can be omitted if necessary and I think this was a good time to not have commas.